src/export/export_snapshot_s3_function.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+import boto3
+
+import constants
+
+
+def lambda_export_rds_snapshot_to_s3(event, context):
+    """start export task of RDS snapshot to S3 bucket"""
+    region = os.environ['Region']
+    rds = boto3.client('rds', region)
+    result = {}
+    export_id = event['identifier'] + constants.EXPORT_TASK_POSTFIX
+    snapshot_id = event['identifier'] + constants.SNAPSHOT_POSTFIX
+    snapshot_arn = check_db_snapshot_is_ready(snapshot_id)
+    account_id = __get_aws_account_id()
+    bucket_name = constants.RDS_SNAPSHOTS_BUCKET_NAME_PREFIX + account_id
+    try:
+        response = rds.start_export_task(
+            ExportTaskIdentifier=export_id,
+            SourceArn=snapshot_arn,
+            S3BucketName=bucket_name,
+            IamRoleArn=os.environ['SNAPSHOT_EXPORT_TASK_ROLE'],
+            KmsKeyId=os.environ['SNAPSHOT_EXPORT_TASK_KEY'],
+        )
+        result['taskname'] = constants.EXPORT_SNAPSHOT
+        result['identifier'] = response['SourceArn']
+        result['status'] = response['Status']
+        return result
+    except Exception as error:
+        raise Exception(error)
+
+
+def check_db_snapshot_is_ready(snapshot_name):
```

Comment:
method name can be something more readable as it returns the arn if the snapshot is available

Reply:
done